README.md

```diff
@@ -556,6 +556,7 @@ Name | Website | Region
 [Remote Garage](/company-profiles/remote-garage.md) | http://www.remotegarage.club/ | India
 [RemoteBase](/company-profiles/remotebase.md) | https://remotebase.com | Worldwide
 [RenoFi](/company-profiles/renofi.md) | https://renofi.com/ |  Worldwide
+[Replit](/company-profiles/replit.md) | https://replit.com/site/careers | Worldwide
```

Comment:
This link should be the main site link, not the careers page

Reply:
Done